### DISCUSSION TITLE
Listener Adding HTML After <body> tag

### DISCUSSION THREAD

**Discussion by hlsamuels** [orchidsoftware/platform#2697] [Q&A]
I'm using 14.0.1 and when my listener triggers it adds new html after the closing body tag of the page. My layouts are all standard Orchid (no blade templates, etc.). Any idea what might be happening?

**Comment by tabuna** [comment]
Maybe you published the views from vendor (In /view/vendor/)?

**Reply by hlsamuels** [reply]
yes, i did..does that cause a problem?
